Easybuoy/storemanager-react
# #163631038 Fix loader error

#### What does this PR do?
Fix loader error.

#### Description of Task to be completed?
- create different types of loader
- change loader types in attendant and product respectively

#### How should this be manually tested?
- Login as an admin
- Navigate to `view products`
- Navigate back to `create sale attendant`
- The form should be showing and not `loader`

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#163631038

#### Screenshots (if appropriate)

#### Questions:
